DQM/SiPixelPhase1Config/python/SiPixelPhase1OfflineDQM_source_cff.py

```diff
@@ -19,12 +19,16 @@
 from DQM.SiPixelPhase1Common.SiPixelPhase1RawData_cfi import *
 #Summary maps
 from DQM.SiPixelPhase1Summary.SiPixelPhase1Summary_cfi import *
+#Additional Efficiency Trend Plots
+from DQM.SiPixelPhase1Track.SiPixelPhase1EfficiencyExtras_cfi import *
+
 #Barycenter plots
 from DQM.SiPixelPhase1Summary.SiPixelBarycenter_cfi import *
 
 from RecoPixelVertexing.PixelLowPtUtilities.ClusterShapeHitFilterESProducer_cfi import *
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 
+
```

Comment:
The cfi is importing the module but SiPixelPhase1EfficiencyExtras is not added to the sequence, is this intentional?

Reply:
Everything in OfflineDQM_source_cff is imported to OfflineDQM_harvesting_cff:
from  DQM.SiPixelPhase1Config.SiPixelPhase1OfflineDQM_source_cff import *

The sequence this module appears in is: siPixelPhase1OfflineDQM_harvesting